src/ui/Form/Checkbox.tsx

```diff
@@ -42,7 +39,7 @@ export default function Checkbox<FormValues extends FieldValues>({
         />
         <Form.CheckboxTick
           role="checkbox"
-          aria-describedby={id + 'error'}
+          aria-describedby={fieldState.invalid ? errMsgId : descId}
```

Comment:
`aria-describedBy` can be used with multiple labels.

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute